### PR TITLE
xplat mdatp docs updates

### PR DESCRIPTION
**Pull request by maximvelichko**
- macOS Intune - modern workflow for app configuration
- macOS Intune - cleaning up profiles
- macOS config - clarifications on exclusions
- Linux config - clarification on exclusions

### CODE REVIEW

**Comment by maximvelichko** [comment]
@AirTudor - please review when you have time

**Comment by mjcaparas** [comment]
Will wait for @AirTudor review and approval.

File: microsoft-365/security/defender-endpoint/mac-install-with-intune.md
Comment (AirTudor) [inline microsoft-365/security/defender-endpoint/mac-install-with-intune.md]: Full product name would be better than just `Defender`

Comment (AirTudor) [reply microsoft-365/security/defender-endpoint/mac-install-with-intune.md]: Same in other places, I don't believe we refer to our product just as `Defender`

File: microsoft-365/security/defender-endpoint/mac-install-with-intune.md
Comment (AirTudor) [inline microsoft-365/security/defender-endpoint/mac-install-with-intune.md]: Minor: I believe this should either be lower-case `c`, or upper-case `P` (i.e. configuration profile / Configuration Profile). Having just Configuration capitalized looks a bit confusing

File: microsoft-365/security/defender-endpoint/mac-preferences.md
Comment (AirTudor) [inline microsoft-365/security/defender-endpoint/mac-preferences.md]: `Mac` instead

File: microsoft-365/security/defender-endpoint/mac-install-with-intune.md
Comment (AirTudor) [inline microsoft-365/security/defender-endpoint/mac-install-with-intune.md]: This should probably be removed from under `Kernel Extensions` (these bundles are for system extensions instead)

**Review by AirTudor — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Comment by maximvelichko** [comment]
@mjcaparas , can you complete/merge it?

**Pull request merged**
